mlflow/store/tracking/file_store.py

```diff
@@ -1274,6 +1281,109 @@ class _SummaryTuple:
             for summary in summaries
         ]
 
+    def _create_promptlab_run(
+        self,
+        experiment_id: str,
+        run_name: str,
+        tags: List[RunTag],
+        prompt_template: str,
+        prompt_parameters: List[Param],
+        model_route: str,
+        model_parameters: List[Param],
+        model_input: str,
+        model_output_parameters: List[Param],
+        model_output: str,
+        mlflow_version: str,
+        user_id: str,
+        start_time: str,
+    ):
+        """
+        Creates a run with the specified attributes.
+        """
```

Comment:
Nit: Can we update the docstring here to explain what promptlab run creation is?